packages/pug/test/pug.test.js

```diff
@@ -979,12 +979,20 @@ describe('pug', function(){
       assert(actual === expected);
     });
     it('should support .pug.compileClient(str, options)', function () {
-      var src = '.bar= self.foo'
+      var src = '.bar= self.foo';
       var fn = pug.compileClient(src, {self: true});
       fn = Function('pug', fn.toString() + '\nreturn template;')(pug.runtime);
       var actual = fn({foo: 'baz'});
       assert(actual === '<div class="bar">baz</div>');
     });
+    it('should support module syntax .pug.compileClient(str, options)', function () {
+      var src = '.bar= self.foo';
+      var fn = pug.compileClient(src, {self: true, module: true, inlineRuntimeFunctions: false});
+      fs.writeFileSync(__dirname + '/temp/input-compileModuleFileClient.js', fn.toString());
```

Comment:
`compileClient` should already return a string, so you shouldn't need to call `.toString()`, other tests may be out of date though.